tests/test_density_matrix.py

```diff
@@ -96,13 +94,91 @@ def test_init_without_data_success(self, n: int) -> None:
     def test_init_with_data_success(self) -> None:
         # don't use rand_dm here since want to check
         for n in range(3):
-            data = randobj.rand_herm(2**n)
+            _data = randobj.rand_herm(2**n)
+
+    def test_init_with_state_sucess(self, fx_rng: Generator) -> None:
+        # both "numerical" statevec and Statevec object
+        # relies on Statevec constructor validation
+
+        nqb = fx_rng.integers(2, 5)
+        print(f"nqb is {nqb}")
+        rand_angles = fx_rng.random(nqb) * 2 * np.pi
+        rand_planes = fx_rng.choice(np.array([i for i in graphix.pauli.Plane]), nqb)
```

Comment:
Is it necessary to cast to `np.ndarray` before choice?

Reply:
No, it wasn't, thanks for having noticed this! Fixed in 7e56152.